terror/terror.go

```diff
@@ -72,6 +72,7 @@ const (
 	ClassTypes
 	ClassGlobal
 	ClassMockTikv
+	ClassJSON
```

Comment:
this is useful for json errors, although they are absent now. so I prefer keep it here, and we will use it later.